client/src/components/Nav/NavMenuIcon.jsx

```diff
@@ -7,29 +7,43 @@ import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
 
 // How to wrap `MenuItem` components inside of `Link` components without warnings?
 // https://github.com/callemall/material-ui/issues/4899
-const NavMenuIcon = () => (
+const NavMenuIcon = (props) => (
   <IconMenu
     iconButtonElement={
       <IconButton>
         <MoreVertIcon />
       </IconButton>
     }
+
     targetOrigin={{ horizontal: 'right', vertical: 'top' }}
     anchorOrigin={{ horizontal: 'right', vertical: 'top' }}
   >
-    <MenuItem
+    {!props.loggedIn?<MenuItem
```

Comment:
Hey, what about those 2 separate components we were discussing the other day? 

* `MenuItem`s for users who are logged in
* another for guest users

Much cleaner that way, and you don't those ugly ternary statements (`props.loggedIn`)